native/index.ts

```diff
@@ -36,8 +36,6 @@ if (isDev) {
   proxyPath = path.join(__dirname, "../target/debug/radicle-proxy");
 
   proxyArgs.push(
-    "--default-seed",
-    "hybz9gfgtd9d4pd14a6r66j5hz6f77fed4jdu7pana4fxaxbt369kg@setzling.radicle.xyz:12345",
```

Comment:
Once we have a working seed again. Will we put it back here?

Reply:
Yes. But then we would set it via the environment variable so that you can still overwrite it from the outside.

Reply:
Gotcha.